lib/hanami/utils/kernel.rb

```diff
@@ -404,15 +413,14 @@ def self.Integer(arg)
       #   # Missing #respond_to?
       #   input = BasicObject.new
       #   Hanami::Utils::Kernel.BigDecimal(input) # => TypeError
-      def self.BigDecimal(arg)
+      def self.BigDecimal(arg, precision: BIGDECIMAL_DEFAULT_PRECISION)
```

Comment:
@jodosha here are the three versions compared (currently released, master, replacing case/when with if/else): https://gist.github.com/pascalbetz/05972b4aab6d5a948c8a

I copied them into one "self-containing" file in order to avoid version troubles with installed gems.

Tested with 2.2.0, with 2.2.2 and jruby. Converting to if/else does not gain us much on jruby, it seems.